README.md

```diff
@@ -1,2 +1,82 @@
-# subspace-chain-alerts
-Important event alerts for subspace blockchains
+## Subspace Chain Alerts (PoC)
+
+Important event alerts for Subspace blockchains.
+
+### Status: Proof of Concept
+
+- This repository is in a PoC state. Interfaces, configuration, and behavior may change without notice.
+- Hardcoded settings are used for speed of iteration (Slack channel, WebSocket URL, thresholds, etc.).
+
+### What it does currently
+
+- Connects to a local Subspace node over WebSocket (`ws://127.0.0.1:9944`).
+- Keeps runtime metadata up to date via a background updater.
+- Posts Slack messages to a test channel when:
+  - The process launches and connects to the node (first observed block).
+  - Any `Sudo::*` extrinsic is included in a best block.
+  - Any `Balances::force*` extrinsic is included.
+  - Any `Balances::*` extrinsic whose transfer value exceeds a high threshold.
```

Comment:
I would prefer to track the available checks on the ticket, rather than repeating them in the README. This list will quickly go out of date, and maintaining it in the README will cause merge conflicts.

We also talk about the first block message below, no need to repeat that in detail here.